integration_tests/kpi.feature

```diff
@@ -35,27 +35,33 @@ Feature: KPI integration tests
      Then self_sufficiency of {'Grid': 1.0} are correctly reported
      And self_consumption of {'Grid': 1.0} are correctly reported
 
-  Scenario Outline: D3ASIM-2103, self sufficiency reports correct values on levels higher than houses
-     Given we have a scenario named <scenario_file>
+  Scenario: D3ASIM-2103, self sufficiency reports correct values on levels higher than houses for SSM
```

Comment:
SSM? What is it?

Reply:
single sided market

Reply:
As I see the acronym in the code the first time i suggest to write "1-sided market" or similar instead of it.

Reply:
There is always the first time for everything, so I think its okay to keep it